thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js

```diff
@@ -31,6 +32,9 @@ export default Ember.Component.extend({
     show: false
   },
 
+  /**
+   * Adding the buildTooltip Template helper to the this context
```

Comment:
nit: "Adding...to **the** **this** context" (typo?)

Reply:
Sorry for the confusion, I meant : => this.buildTooltip.
The tooltip callback function doesn't have access to the global context, but has access to the `this`, I'm setting it on the `this` context